faculty/models.py

```diff
@@ -0,0 +1,14 @@
+from django.db import models
+from account.models import Lecturer
+
+# Create your models here.
+
+class Faculty(models.Model):
+    dept_name = models.CharField(max_length=255, blank=True, null=True)
+    info_description = models.TextField(blank=True, null=True)
+    images = models.ImageField(upload_to='faculty_images/', blank=True, null= True,)
+    other_titles = models.CharField(blank=True, null=True, max_length=50)
+    
+    reps_name = models.OneToOneField(
```

Comment:
There can be more than one instance of a Lecturer as a rep. Also, Students can be rep too.